libcxx/test/libcxx/atomics/atomics.ref/assert.compare_exchange_strong.pass.cpp

```diff
@@ -0,0 +1,63 @@
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: has-unix-headers
+// UNSUPPORTED: c++03, c++11, c++14, c++17
+// UNSUPPORTED: libcpp-hardening-mode=none
```

Comment:
This should be `UNSUPPORTED: libcpp-hardening-mode=none || libcpp-hardening-mode=fast`

Reply:
Done